temporalio/worker/worker.py

```diff
@@ -206,6 +209,7 @@ def __init__(
             graceful_shutdown_timeout=graceful_shutdown_timeout,
             shared_state_manager=shared_state_manager,
             debug_mode=debug_mode,
+            use_worker_versioning=use_worker_versioning,
```

Comment:
You also need to pass this to `temporalio.bridge.worker.WorkerConfig` below

Reply:
It is?